components/alert/src/Alert.js

```diff
@@ -11,93 +11,98 @@ import { CSSTransition } from 'react-transition-group';
 // Component
 // ==============================
 
-export const Alert = ({ appearance, isClosable, icon: Icon, children }) => {
+export const Alert = ({ appearance, closable, icon: Icon, children }) => {
+	const { COLORS } = useTheme();
+	const mq = useMediaQuery();
 	const [open, setOpen] = useState(true);
-	const { breakpoints, alert } = useTheme();
-	const mq = paint(breakpoints);
 
 	const iconMap = {
 		success: TickIcon,
-		information: InfoIcon,
+		info: InfoIcon,
 		warning: AlertIcon,
 		danger: AlertIcon,
 		system: AlertIcon,
 	};
 
-	// Set default icon
+	// Set a default icon
 	if (Icon === undefined) {
 		Icon = iconMap[appearance];
 	}
 
-	const style = {
-		common: {
-			marginBottom: alert.marginBottom,
-			padding: [
-				alert.padding,
-				isClosable ? `${alert.padding} 3rem ${alert.padding} ${alert.padding}` : alert.padding,
-			],
-			position: 'relative',
-			display: [null, 'flex'],
-			zIndex: 1,
-			transition: 'opacity 0.3s ease-in-out',
-
-			'a, h1, h2, h3, h4, h5, h6, ol, ul': {
-				color: 'inherit',
-			},
-
-			'&.anim-exit-active': {
-				opacity: 0,
-			},
-		},
-		appearance: {
-			color: alert.appearance[appearance].color,
-			backgroundColor: alert.appearance[appearance].backgroundColor,
-			borderTop: `${alert.borderWidth} solid`,
-			borderBottom: `${alert.borderWidth} solid`,
-			borderColor: alert.appearance[appearance].borderColor,
-		},
-		close: {
+	// Common styling
+	const styleCommon = {
+		marginBottom: '1.3125rem',
+		padding: ['1.125rem', closable ? `1.125rem 1.875rem 1.125rem 1.125rem` : '1.125rem'],
+		position: 'relative',
+		display: [null, 'flex'],
+		zIndex: 1,
+		transition: 'opacity 0.3s ease-in-out',
+
+		'a, h1, h2, h3, h4, h5, h6, ol, ul': {
 			color: 'inherit',
-			position: ['relative', 'absolute'],
-			zIndex: 1,
-			float: ['right', 'none'],
-			top: '0.3rem',
-			right: 0,
-			marginTop: ['-1.8rem', 0],
-			marginRight: ['-1.8rem', 0],
-			opacity: 1,
-
-			':hover': {
-				opacity: 0.5,
-			},
 		},
-		icon: {
-			float: ['left', 'none'],
-			marginRight: ['0.6rem', '1.2rem'],
-			flex: 'none',
-		},
-		body: {
-			position: 'relative',
-			flex: 1,
-			top: [null, Icon && '0.2rem'],
+
+		'&.anim-exit-active': {
+			opacity: 0,
 		},
 	};
 
+	// Appearance styling
+	const styleAppearance = {
+		color: appearance === 'system' ? 'black' : COLORS[appearance],
+		backgroundColor: appearance === 'system' ? COLORS.system : COLORS.tints[`${appearance}5`],
+		borderTop: '1px solid',
+		borderBottom: '1px solid',
+		borderColor: appearance === 'system' ? COLORS.system : COLORS.tints[`${appearance}50`],
+	};
+
 	return (
 		<CSSTransition in={open} unmountOnExit classNames="anim" timeout={400}>
-			<div css={mq({ ...style.common, ...style.appearance })}>
-				{isClosable && (
+			<div css={mq({ ...styleCommon, ...styleAppearance })}>
```

Comment:
You can also _merge_ the objects with emotion's array syntax 

```
<div css={mq([ styleCommon, styleAppearance ])}>
```

However the result is a little different, common styles in both common and appearance get both applied, whereas with the spread syntax, new declarations on the same property replace the previous one (which is what you want here I think!)

Reply:
Yeah I hate seeing CSS overrides within the same class

Reply:
Hmm... according to this shouldn't the array syntax merge them?

https://emotion.sh/docs/composition